domains/akshit.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Akshit's Personal Website",
+    "repo": "https://github.com/4kshi7/yzy-discography",
+    "owner": {
+        "username": "4kshi7",
+        "email": "akshitgaur2003@gmail.com"
+    },
+    "record": {
+        "CNAME": "https://yzy-2k.vercel.app/"

```

CNAME should not contain protocol
```suggestion
        "CNAME": "yzy-2k.vercel.app"

```